source/blog/2016-05-19-introducing-subteams.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Introducing Subteams
+author: Tom Dale
+tags: Recent Posts
+---
+
+I'm very happy to share with you our plan to scale Ember's day-to-day
+decision-making and coordination–something we're calling Subteams.
```

Comment:
Seems good to mention that this is a follow up to the announcement in the keynote, or a more detailed outline of what you and Yehuda briefly touched on in the keynote, something like that. Otherwise I suspect people who attended or saw the video will read this and be confused by it being an "introduction". More importantly they probably won't continue reading, and there are bits of new useful info in the rest of the post.